.travis.yml

```diff
@@ -1,13 +1,13 @@
 language: elixir
-branches:
-  only:
-    - "master"
+
 cache:
   directories:
     - deps
     - _build
 matrix:
   include:
+    - elixir: 1.9.4
```

Comment:
This is a good thing to do but it doesn't seem directly related to the snappyer change.  If you want to make this change, please make a separate pull request.